helix-view/src/editor.rs

```diff
@@ -92,6 +94,7 @@ impl Default for Config {
             idle_timeout: Duration::from_millis(400),
             completion_trigger_len: 2,
             auto_info: true,
+            git_ignore: false,
```

Comment:
I don't know where @pickfire's comment went but this should indeed be true to match the current default.

Reply:
Okay, thank you for your note.

I am seeing with this, and also the next comment as saying you'd like to define defaults based on the `ignore` crate, specifically leaving in the default filters.